papyri/myst_serialiser.py

```diff
@@ -54,16 +56,20 @@ def serialize(instance, annotation):
         if hasattr(ann_, "type"):
             type_ = ann_.type
         if isinstance(serialized_data, dict):
-            return {**serialized_data, "type": type_}
-        return {"data": serialized_data, "type": type_}
+            return {**serialized_data, "type": type_, "key": uuid.uuid4().hex}
+        return {"data": serialized_data, "type": type_, "key": uuid.uuid4().hex}
```

Comment:
That would make the json different each time we serialize, would it not? I'd like to not rely on arbitrary number to do that. We should try to be as predictable as  possible.

If that's just temporary feel free to merge but to add a issue to remember to remove it.

Reply:
Yeah, completely agree with that. I was just trying to get something unique to add html to the unique div, while rendering, it is definitely temporary.

Reply:
You probably can generate in the templates if needs be.